### DISCUSSION TITLE
Lazer crash on start

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#23705] [Q&A]
### Type

Crash to desktop

### Bug description

As the title stated, the osuLazer crash on start after seeing the opening animation, usually when the little Osu icon bounces from the text.

Please kindly find the attached logs with this post.
If there is any information I can provide, please let me know.

Version: `app-2023.513.0`

### Version

app-2023.513.0

### Logs

[performance.log](https://github.com/ppy/osu/files/11591478/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11591479/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11591480/updater.log)
[input.log](https://github.com/ppy/osu/files/11591481/input.log)
[events.log](https://github.com/ppy/osu/files/11596546/events.log)

Due to Github does not support uploading `.evtx` files, I have changed the extension from `.evtx` to `.log`
Please rename the file extension to `.evtx`

The last crash(when the post being created) was about `2023-05-29 11:50:57`

**Comment by bdach** [comment]
Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Comment by LIII-Works** [comment]
[events.log](https://github.com/ppy/osu/files/11596546/events.log)

Due to Github does not support uploading `.evtx` files, I have changed the extension from `.evtx` to `.log`
Please rename the file extension to `.evtx`

The last crash(when the post being created) was about `2023-05-29 11:50:57`

Thank you

**Comment by bdach** [comment]
The attached event log doesn't seem to have any entries associated with osu!. Nor any entries between `2023-05-29 11:40:23.313356` and `2023-05-29 11:54:53.180233`.

It does however contain a whole bunch of errors from `NahimicSvc32.exe`. Google tells me it seems to be some sort of audio enhancement program from MSI or something. I would probably start by seeing if that has any updates. Or killing it.

**Comment by LIII-Works** [comment]
Thank you for the insight. I have disabled the Nahimic service in Service and the Lazer can operate normally now.

I have a follow-up question, why or how osuLazer is tightened with an audio device/driver? I rarely see software crashes caused by an audio driver error. Just curious

**Reply by bdach** [reply]
I'm not sure why lazer is the sensitive one to that (may be due to bass internals, may be something else entirely to do with our threading model), but to me it's more of a broken driver situation rather than a "lazer is sensitive" situation. The driver in question was really spewing _way_ too many errors in the event log attached, I'd like to think.
